nuxeo-common/src/test/java/org/nuxeo/common/utils/TestTextTemplate.java

```diff
@@ -157,4 +158,32 @@ public void testEscapeVariable() {
         assertEquals("{$$CZjbsiX748UF583qkbinsQ==}", templates.processText("{$$CZjbsiX748UF583qkbinsQ==}"));
     }
 
+    @Test
+    public void testFreetemplateFileNameValidity() throws FileNotFoundException, IOException, TemplateException {
+        final TextTemplate textTemplate = new TextTemplate();
```

Comment:
We don't use `final` for local variables (or method parameters), as this adds noise for the reader.

Reply:
Using final is a good practice to me, and documented as this.
Is it a blocker to you ?